freqtrade/freqai/freqai_interface.py

```diff
@@ -562,6 +563,9 @@ def extract_data_and_train_model(
             self.dd.pair_to_end_of_training_queue(pair)
         self.dd.save_data(model, pair, dk)
 
+        if self.freqai_info["feature_parameters"].get("plot_feature_importance", True):
```

Comment:
should the default be False?
considering it's also not part of the docs (should probably be fixed ...) - people won't know to look for this - which makes the generated plot missed CPU cycles.

This param should also be added to the json schema (`constants.py`) ... so we know we'll only get a boolean through.

Reply:
What about this point? It made sense to me - just didn’t get around to it yet.

Reply:
Good catch - overlooked this missing point.
added in ea58c29ded45033eb175d51f7041facb3ec33fa7.